packages/deploy-script-support/src/extract-proposal.js

```diff
@@ -240,22 +236,24 @@ export const extractCoreProposalBundles = async (
   harden(bundles);
 
   // Extract the manifest references and calls.
-  const makeCPArgs = extracted.map(({ ref, call, overrideManifest }) => ({
+  const metadataRecords = extracted.map(({ ref, call, customManifest }) => ({
     ref,
     call,
-    overrideManifest,
+    customManifest,
   }));
-  harden(makeCPArgs);
+  harden(metadataRecords);
 
   const code = `\
-// This is generated by @agoric/cosmic-swingset/src/extract-proposal.js - DO NOT EDIT
+// This is generated by @agoric/deploy-script-support/src/extract-proposal.js - DO NOT EDIT
 /* eslint-disable */
 
-const makeCoreProposalArgs = harden(${stringify(makeCPArgs, true)});
-
-const makeCoreProposalBehavior = ${makeCoreProposalBehavior};
+const metadataRecords = harden(${stringify(metadataRecords, true)});
 
-(${makeEnactCoreProposals})({ makeCoreProposalArgs, E });
+// Make an enactCoreProposals function and "export" it by way of script completion value.
+((
+  makeCoreProposalBehavior = ${makeCoreProposalBehavior},
+  makeEnactCoreProposals = ${makeEnactCoreProposals},
+) => makeEnactCoreProposals({ metadataRecords, E }))();
```

Comment:
Wow!  I found this really tricky to understand, but eventually puzzled through it... I take it the IFFE default arguments are used just to create local bindings?

I'm not sure that's as clear as creating them at the top level.  Would you be able to write a very clear explanation of what these lines are trying to do?

If it's too hard to render in English, I'd prefer something like:

```suggestion
const makeCoreProposalBehavior = ${makeCoreProposalBehavior};

const makeEnactCoreProposals = ${makeEnactCoreProposals};

// Make an enactCoreProposals function and "export" it by way of script completion value.
const enactCoreProposals = makeEnactCoreProposals({ metadataRecords, E });
enactCoreProposals;
```